snap/hooks/configure

```diff
@@ -39,6 +40,20 @@ then
   systemctl restart snap.${SNAP_NAME}.daemon-apiserver
 fi
 
+# Enable the aggregation layer (continue)
```

Comment:
Why do we split these in two sections?

Reply:
To make sure the fix applies on upgrade from the version where `--requestheader-client-ca-file` has been already added.